packages/xo-web/src/xo-app/home/host-item.js

```diff
@@ -262,6 +273,17 @@ export default class HostItem extends Component {
         })
       }
 
+      if (mdadmHealth?.raid?.State && !['clean', 'active'].includes(mdadmHealth.raid.State)) {
```

Comment:
Always explicit condition. Exepct for `boolean`
```suggestion
      if (mdadmHealth?.raid?.State !== undefined && !['clean', 'active'].includes(mdadmHealth.raid.State)) {
```